be/src/olap/rowset/segment_v2/binary_dict_page.cpp

```diff
@@ -257,23 +257,33 @@ Status BinaryDictPageDecoder::next_batch(size_t* n, vectorized::MutableColumnPtr
     const int32_t* data_array = reinterpret_cast<const int32_t*>(_bit_shuffle_ptr->_chunk.data);
     size_t start_index = _bit_shuffle_ptr->_cur_index;
 
-    // todo(wb) support nullable
-    if (dst->is_predicate_column()) {
+    auto* dst_col_ptr = dst.get();
+    if (dst->is_nullable()) {
+        auto nullable_column = assert_cast<vectorized::ColumnNullable*>(dst.get());
+        dst_col_ptr = nullable_column->get_nested_column_ptr().get();
+        
+        // fill null bitmap here, not null;
+        for (int i = 0; i < max_fetch; i++) {
+            nullable_column->get_null_map_data().push_back(0);
+        }
+    }
+
+    if (dst_col_ptr->is_predicate_column()) {
```

Comment:
dst_col_ptr->is_predicate_column() is `true` means the column is `PredicateColumnType` why `dst_col_prt` convert to vectorized::ColumnStringValue*

Reply:
Because current class is ```binary_dict_page```, only ```ColumnStringValue``` will enter this method.